src/base/bittorrent/session.cpp

```diff
@@ -4352,15 +4357,17 @@ void Session::createTorrentHandle(const lt::torrent_handle &nativeHandle)
         // The following is useless for newly added magnet
         if (!fromMagnetUri) {
             // Backup torrent file
-            const QDir resumeDataDir(m_resumeFolderPath);
-            const QString newFile = resumeDataDir.absoluteFilePath(QString("%1.torrent").arg(torrent->hash()));
-            if (torrent->saveTorrentFile(newFile)) {
+            const QDir resumeDataDir {m_resumeFolderPath};
+            const QString torrentFileName {QString {"%1.torrent"}.arg(torrent->hash())};
+            try {
+                torrent->saveTorrentFile(resumeDataDir.absoluteFilePath(torrentFileName));
```

Comment:
This call needs to be updated too (hence the failing CI), if you haven't already done so
`torrent->info().saveToFile(resumeDataDir.absoluteFilePath(torrentFileName));`

Reply:
I just forgot to click "Save" button after change it...

Reply:
Happens!

Reply:
Fixed!